test_util/test_runner.py

```diff
@@ -38,12 +39,15 @@ def integration_test(
             the test environment
         pytest_dir: directory of test package on test host
         pytest_cmd: string representing command for py.test
+        test_dockercfg_hook: if set to True, we enable the remove dockercfg hook
+            module
```

Comment:
I know I said earlier that we _could_ put this in this function argument, but we are going to want to control this from ci and between variants, so instead lets propagate this with TEST_ADD_ENV_*

To do this locally, make no edits to test_util/test_runner.py and just run this as TEST_ADD_ENV_DOCKERCFG_HOOK='true' ccm-deploy-test

To do this in CI, you just add another param here:
https://teamcity.mesosphere.io/admin/editBuildParams.html?id=template:ClosedSource_Dcos_IntegrationTests_CloudIntegrationTests_DcosOssCcmInstallTest
(ill go ahead and populate that in a moment, but FYI so you can toggle it)